pkgs/development/compilers/ghc/8.0.1.nix

```diff
@@ -57,6 +57,8 @@ stdenv.mkDerivation rec {
   stripDebugFlags = [ "-S" ] ++ stdenv.lib.optional (!stdenv.isDarwin) "--keep-file-symbols";
 
   postInstall = ''
+    paxmark m $out/lib/${name}/bin/{ghc,haddock}
```

Comment:
`paxmark` is defined as a stub function which is overridden by `paxctl`'s setup-hook; on non-linux `paxmark` is a no-op.